### DISCUSSION TITLE
[@types/http-signature] _RequestSigner.writeDataHeader should be _RequestSigner.writeDateHeader

### DISCUSSION THREAD

**Discussion by karvapallo** [DefinitelyTyped/DefinitelyTyped#67022] [Issues with a @types package] [answered]
The typings have a misspelled method name. The typo in question is called `writeDataHeader` in the `_RequestSigner` class. The method name is `writeDateHeader` NOT `writeDataHeader`

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
Oops. Just opened a PR: #67025
